Staceyann_King_submission/src/sortedArr.js

```diff
@@ -0,0 +1,28 @@
+function sortedArr(array, target){
+  
+    for(let i = 0; i < array.length; i++){
+        let j = i+1
+        if(array[i] == target){
+            return i
+        }
+        else if(target > array[i] && target < array[j]){
+            return j
+        }
+    }
+   
+    let first_index_of_the_array = 0
+    let last_index_of_the_array = array.length-1
```

Comment:
Technically, since you aren't modifying these variables, I think they should be `const`'s instead of `let`s